Mathlib/Tactic/LeftRight.lean

```diff
@@ -0,0 +1,51 @@
+import Lean
+import Lean.Meta
+import Lean.Elab
+
+namespace Mathlib.Tactic.Left
+open Lean Meta Elab
+open Tactic
+def leftMeta (mvarId : MVarId) : MetaM (List MVarId) := do
+  withMVarContext mvarId do
+    checkNotAssigned mvarId `left
+    let target ← getMVarType' mvarId
+    matchConstInduct target.getAppFn
+      (fun _ => throwTacticEx `constructor mvarId "target is not an inductive datatype")
+      fun ival us => do
+        unless ival.ctors.length == 2 do
+          throwTacticEx `constructor mvarId "left target applies for inductive types with exactly two constructors"
+        let ctor := ival.ctors.toArray[0]
```

Comment:
Considering that the only difference between these two functions is the index of `toArray`, I think it's better to extract the common code, which receives a `Bool` and uses `0` if `false` and vice-versa

Reply:
I agree. Will do this.

Reply:
Done in d49c0d8555d660175de52999